apps/token-e2e/src/support/wallet-vega.functions.js

```diff
@@ -15,3 +15,15 @@ Cypress.Commands.add(
       });
   }
 );
+
+Cypress.Commands.add('connectPublicKey', function (pubKey) {
```

Comment:
suggestion (non blocking): Perhaps we can move this to libs/cypress for use in other e2e projects

Reply:
Done!